client/src/containers/Gene/GeneProfile.js

```diff
@@ -144,55 +168,43 @@ class GeneProfile extends Component {
               </div>
           }
         </PageRight>
-        {
-          this.state.data ?
-            <KillGeneDialog
-              geneName={this.state.data.cgcName}
-              open={this.state.showKillGeneDialog}
-              onClose={this.closeKillGeneDialog}
-              onSubmitSuccess={(data) => {
-                this.setState({
-                  data: data,
-                }, () => {
-                  this.closeKillGeneDialog();
-                });
-              }}
-            /> :
-            null
-        }
-        {
-          this.state.data ?
-            <MergeGeneDialog
-              geneName={this.state.data.cgcName}
-              open={this.state.showMergeGeneDialog}
-              onClose={this.closeMergeGeneDialog}
-              onSubmitSuccess={(data) => {
-                this.setState({
-                  data: data,
-                }, () => {
-                  this.closeMergeGeneDialog();
-                });
-              }}
-            /> :
-            null
-        }
-        {
-          this.state.data ?
-            <SplitGeneDialog
-              geneName={this.state.data.cgcName}
-              biotypeOriginal={this.state.data.biotype}
-              open={this.state.showSplitGeneDialog}
-              onClose={this.closeSplitGeneDialog}
-              onSubmitSuccess={(data) => {
-                this.setState({
-                  data: data,
-                }, () => {
-                  this.closeSplitGeneDialog();
-                });
-              }}
-            /> :
-            null
-        }
+        <KillGeneDialog
```

Comment:
Could the name used be either $state.cgcName or $state.sequenceName?

Reply:
Good catch! Gonna fix this.